java/pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>com.agrirouter.proto</groupId>
     <artifactId>agrirouter-tmt-protobuf-definitions</artifactId>
-    <version>2.1.0-SNAPSHOT</version>
+    <version>2.2.0-SNAPSHOT</version>
```

Comment:
Since the version is changing only within the release process I would recommend not changing the version within the PR.

Reply:
Fixed